crates/red_knot_python_semantic/src/types/signatures.rs

```diff
@@ -1023,25 +1023,4 @@ mod tests {
         // With no decorators, internal and external signature are the same
         assert_eq!(func.signature(&db), &expected_sig);
     }
-
-    #[test]
-    fn external_signature_decorated() {
```

Comment:
If someone thinks that we need a replacement for this test, please let me know. The distinction between internal and external signature might still make sense for things like `@classmethod` that we handle by just flipping a bit.